<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>831</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(136, 138, 133);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="powerBtn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/imgs/powerbutton2.png</normaloff>:/imgs/powerbutton2.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="powerIndicator">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255,255,255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="downBtn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>390</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/imgs/downButton.png</normaloff>:/imgs/downButton.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="upBtn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>280</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/imgs/upButton.png</normaloff>:/imgs/upButton.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="intLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>350</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="text">
      <string>INT</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="checkBtn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>470</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/imgs/checkMark.png</normaloff>:/imgs/checkMark.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>146</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="TwentySession">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Resources.qrc">
          <normaloff>:/imgs/TwentyMinsPieChart.png</normaloff>:/imgs/TwentyMinsPieChart.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="FortyFiveSession">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Resources.qrc">
          <normaloff>:/imgs/FortyFivePieChart.png</normaloff>:/imgs/FortyFivePieChart.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="UserDesignatedSession">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Resources.qrc">
          <normaloff>:/imgs/UserDesignatedSession.png</normaloff>:/imgs/UserDesignatedSession.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>200</y>
       <width>81</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>221</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="Row_Level8">
         <item>
          <widget class="QLabel" name="lvl8">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255,255,255);</string>
           </property>
           <property name="text">
            <string>8</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="Row_Level7">
         <item>
          <widget class="QLabel" name="lvl7">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255,255,255);</string>
           </property>
           <property name="text">
            <string>7</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="Row_Level6">
         <item>
          <widget class="QLabel" name="lvl6">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255,255,255);</string>
           </property>
           <property name="text">
            <string>6</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="Row_Level5">
         <item>
          <widget class="QLabel" name="lvl5">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255,255,255);</string>
           </property>
           <property name="text">
            <string>5</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="Row_Level4">
         <item>
          <widget class="QLabel" name="lvl4">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255,255,255);</string>
           </property>
           <property name="text">
            <string>4</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="Row_Level3">
         <item>
          <widget class="QLabel" name="lvl3">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255,255,255);</string>
           </property>
           <property name="text">
            <string>3</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="Row_Leve2">
         <item>
          <widget class="QLabel" name="lvl2">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255,255,255);</string>
           </property>
           <property name="text">
            <string>2</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="Row_Level1">
         <item>
          <widget class="QLabel" name="lvl1">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255,255,255);</string>
           </property>
           <property name="text">
            <string>1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>340</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="SessionType1">
        <property name="text">
         <string>type 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SessionType2">
        <property name="text">
         <string>type 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SessionType3">
        <property name="text">
         <string>type 3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SessionType4">
        <property name="text">
         <string>type 4</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>20</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Battery: </string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveBtn">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>460</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Treatment</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DampenEars">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>70</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Damp Ears</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ConnectEarclips">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>100</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect Earclips</string>
     </property>
    </widget>
    <widget class="QLabel" name="EarclipsConnected">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>100</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(239, 41, 41);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="EarsDampened">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>70</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(239, 41, 41);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="remainingTimeLabel">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Remaining Session Time: </string>
     </property>
    </widget>
    <widget class="QLabel" name="connectionQuality">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>140</y>
       <width>241</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Connection Quality: No Connection</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>833</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
